Library/Homebrew/env_config.rb

```diff
@@ -19,9 +19,10 @@ module EnvConfig
         description: "Automatically check for updates once per this seconds interval.",
         default:     300,
       },
-      HOMEBREW_BAT:                       {
-        description: "If set, use `bat` for the `brew cat` command.",
-        boolean:     true,
+      HOMEBREW_BAT_CONFIG_PATH:           {
```

Comment:
Keep the existing `HOMEBREW_BAT` variable and allow this to specify just the configuration path.

Reply:
That's exactly what I did in the original commit 😅

Reply:
@MikeMcQuaid Iv'e done this, but surely just allowing the existing `HOMEBREW_BAT` to contain the value of the config path is simpler than the user having to have both variables in their `rc` file, along with less code…
~~~ sh
HOMEBREW_BAT=1
HOMEBREW_BAT_CONFIG_PATH=$(bat --config-file)
~~~

Reply:
`HOMEBREW_BAT` is a documented variable set by users. We don't want it to stop working.

Reply:
Sure, but currently it's just set to `1` (or a truthy value)… Unless I'm missing something, changing the value to a path still achieves exactly the same thing, so I don't see how that would break anything?

Reply:
It's confusing to change the expected input under users feet. Let's just leave it as two variables, please.